packages/js/src/ui/components/primitives/Button.tsx

```diff
@@ -17,7 +17,7 @@ export const buttonVariants = cva(
       },
       size: {
         none: '',
-        icon: 'p-1',
```

Comment:
oopsie

Reply:
We can probably add some tailwind config option for the intellisense to work into these strings as well.
https://cva.style/docs/getting-started/installation#intellisense this is nto specific to the library